### DISCUSSION TITLE
İlk 1000 de miyim?

### DISCUSSION THREAD

**Discussion by THECROVV** [IBAX-io/go-ibax#1649] [Q&A] [answered]
Selamlar herkese ilk 1000 kişi arasına girdiğimi gösteren bir yer var mı acaba bir türlü bulamadım.

**Comment by Bulent3407** [comment, marked as answer]
Gelişim ve onur geliştiricileri menüden

**Reply by THECROVV** [reply]
Teşekkürler

**Reply by Mrtdlk23** [reply]
👍
